### PR TITLE
New CheatSheet: Turkish

### PR DESCRIPTION
**Pull request by ozdemirburak**
**What does your Instant Answer do?**
 Provides a quick reference of beginner phrases for travelers in Turkey.

**What is the data source for your Instant Answer? (Provide a link if possible)**
http://www.linguanaut.com/english_turkish.htm

**Why did you choose this data source?**
Accurate and saw that it was previously used by German cheat sheet.

**Are there any other alternative (better) data sources?**
Nope, as far as I know. However, as a native Turkish speaker, I've changed some of the translations provided by the source.

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
People traveling to Turkey.

**Is this Instant Answer connected to a DuckDuckHack [Instant Answer idea](https://duck.co/ideas)?**
Nope.

**Are you having any problems? Do you need our help with anything?**
Nope.

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**
turkish-cheet-sheet-ss-1

-----
IA Page: https://duck.co/ia/view/turkish_cheat_sheet

### CODE REVIEW

**Comment by moollaza** [comment]
@ozdemirburak thanks a lot for contributing, we really appreciate it! We'll be sure to review this shortly and give you some feedback!

I'll have one of our community Translation Managers sign off on the content as well :+1:

File: share/goodie/cheat_sheets/json/turkish.json [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+    "sourceUrl" : "http://www.linguanaut.com/english_turkish.htm"
+  },
+  "section_order" : [
+    "Basics",
+    "Etiquette",
+    "Getting Help",
+    "Travelling",
+    "Going out for dinner"
+  ],
+  "description" : "Basic Turkish words and phrases",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/turkish.json]: nit: can we move this up below the `name`? This is only a convention, but it helps me (and others) to quickly find the content where I expect it to be :)

File: share/goodie/cheat_sheets/json/turkish.json [hunk 2]
```diff
@@ -0,0 +1,224 @@
+{
+  "id" : "turkish_cheat_sheet",
+  "name" : "Turkish Cheat Sheet",
+  "metadata" : {
+    "sourceName" : "Linguanaut",
+    "sourceUrl" : "http://www.linguanaut.com/english_turkish.htm"
+  },
+  "section_order" : [
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/turkish.json]: we *just* added support for Cheat Sheet templates (documentation will be in next week!)

can you add `"template_type": "reference",` here. The result will make this ***much*** easier to read:

turkish_cheat_sheet_at_duckduckgo

**Comment by ozdemirburak** [comment]
@moollaza Updated according to your comments.

**Comment by ozdemirburak** [comment]
@moollaza ping, any updates on this?

**Comment by moollaza** [comment]
@ozdemirburak sorry for the delay, I'll be putting the language cheat sheets onto beta.duckduckgo.com very soon and we'll then solicite feedback from the community on whether or not they approve of the translations. Once that's done and we've gathered some feedback, we'll need to implement any necessary changes and then this will be ready for merge.

Seeing as I'm not a Turkish speaker, and we don't have any Turkish staff or Translation Mangers we'll need to rely on the community to ensure everyone is happy with the translations here.

**Comment by tagawa** [comment]
[Feedback from Twitter](https://twitter.com/esgalya/status/644826042188873728):
"As a native Turkish speaker, nothing seems wrong."

**Comment by moollaza** [comment]
@tagawa thanks! I'm happy with that :+1:

**Comment by moollaza** [comment]
@ozdemirburak this LGTM :+1:

Thanks a lot for contributing (we really appreciate it :smiley:). We invite you to stick around and comment/help on other PR's & Issues or even submit an Instant Answer!

**Comment by ozdemirburak** [comment]
@moollaza thank you, already submitted drinks pr to spice repository though #2191 :)

**Comment by moollaza** [comment]
@ozdemirburak congrats! It's live: https://duckduckgo.com/?q=turkish+cheat+sheet&ia=cheatsheetl

Thanks again for taking the time to contribute! We really appreciate it. Feel free to stick around and comment/help on other PR's, tackle open issues or even submit more Instant Answers!

**Pull request merged**